README.md

```diff
@@ -18,115 +19,20 @@ See live examples of GOV.UK Frontend components, and guidance on when to use
 them in your service, in the [GOV.UK Design
 System](https://govuk-design-system-production.cloudapps.digital/).
 
-## How to use GOV.UK Frontend
```

Comment:
Are we sure about moving the npm instructions into a different document?

Majority of packages include their install instructions as quickly as possible, it feels with this change we're making it harder to find.

Reply:
I *think* moving them makes sense so that the README stays a manageable length. If we have the "install with npm" instructions in the README we should also have the "installing from dist" instructions there and together that is a lot of text. I guess we could have a table of contents as an overview to help user navigate it if it was a longer file... 🤔 But I think we could take this to user testing and see how it goes. 

This change was initiated by @36degrees, he might have some thoughts on this too.

Reply:
We mobbed on this with the team and the decision was that as long as the instructions are signposted clearly from `README`, this shouldn't be a problem. (Also: it was mentioned that when `install-from-npm` and `install-from-dist` were ran back to back in the `README`, some users were not clear where one ended and the next one started. Moving them into their own docs solves this.)